applications/ila-cookie-banner/RELEASE.md

```diff
@@ -0,0 +1,98 @@
+
+### Step 1. Create the new release folder
+
++ Create a new folder, named for the versioned release, at the root of `onetrust.techservices.illinois.edu`
+
+	> For example, if `1.0.0`: https://onetrust.techservices.illinois.edu/1.0.0
+		
+### Step 2. Get the latest cookie banner zip file from GitHub Actions
+
++ Visit https://github.com/app-illinois/Design-Resources/actions
++ Click on [`Cookie banner zip`](https://github.com/app-illinois/Design-Resources/actions/workflows/cookie-banner-zip.yaml)
++ Click on the the latest successful run - the top one with a green icon.
+
+	> Warning: Failed runs are not expected. Contact the development team if the most recent `Cookie banner zip` GitHub action failed (indicated by a red icon).
+
++ Find and click `cookie-banner` under the `Artifacts` section of the `GitHub Actions Cookie banner zip` page.
++ Find the Zip file in your browser downloads directory.
+	
+### Step 3. Extract & upload the contents of the zip file to the path created in step one
+
+After the zip file contents are extracted & uploaded, the contents of the `1.0.0` folder should look like:
+
+	1.0.0
+	├── css
+	│   ├── ila-cookie-banner.css
+	│   ├── ila-cookie-uic-colors.css
+	│   ├── ila-cookie-uis-colors.css
+	│   └── ila-cookie-uiuc-colors.css
+	├── js
+	│   └── ila-cookie-banner.js
+	├── otSDKStub.js
+	├── partials
+	│   └── ila-cookie-banner-content.part.html
+	└── RELEASE.md
+
+### Step 4. Verify the main JavaScript file
+
++ With a text editor, open `otSDKStub.js`
++ For release `1.0.0`, verify that the first line of `otSDKStub.js` contains `1.0.0`:
+	
+	```javascript
+	let cookie_url = 'https://onetrust.techservices.illinois.edu/1.0.0';
+	```
+	
+	> Warning: If the version number (such as `1.0.0`) in the first line does not match the expected release version, the new version will not load properly. Reach out to the development team.
+	
+## Production Release
+
+### Step 5. Activate the New Release
+
++ Make a backup copy of `https://onetrust.techservices.illinois.edu/otSDKStub.js` to `YYYY.MM.DD.otSDKStub.js`
+
+	> Warning: The next step causes the new version of the cookie banner to start appearing across all sites that use the file.
+
++ Copy `otSDKStub.js` from `1.0.0` to the CDN root. 
+ 
+	> Tip: The file `otSDKStub.js` in the CDN root determines which version of the cookie notice is activated.
+
+	> For example: copy `https://onetrust.techservices.illinois.edu/1.0.0/otSDKStub.js` to `https://onetrust.techservices.illinois.edu/otSDKStub.js`
+
+### Step 6. Verify the Release
+
++ In a browser, visit a website that imports `https://onetrust.techservices.illinois.edu/otSDKStub.js` with a `<script src=...>` line. For example, we have been testing with https://cookieme.rslater.web.illinois.edu/
+
++ Press F12 to open the `Developer Tools`.
++ Find and open the `Network` tab of `Developer Tools`.
++ Ensure that `Disable Cache` is checked. (Otherwise the browser may re-use old downloaded files to save time, but break this test.)
```

Comment:
```suggestion
+ Ensure that `Disable Cache` is checked (Otherwise the browser may re-use old downloaded files to save time, but break this test)
```